test/common/numa_helpers.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef UMF_NUMA_HELPERS_HPP
+#define UMF_NUMA_HELPERS_HPP 1
+
+#include <gtest/gtest.h>
+#include <numa.h>
+#include <numaif.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "test_helpers.h"
+
+// returns the node where page starting at 'ptr' resides
+static inline void getNumaNodeByPtr(void *ptr, int *node) {
```

Comment:
I think you could describe in commit message (for future reference) why you changed this function to return void

Reply:
Added link to gtest documentation